.github/actions/run-backend-tests/action.yml

```diff
@@ -2,29 +2,28 @@
 # This is a composite action that packages our backend Django tests.
 # It is called by the `ci-backend.yml` job using a matrix.
 #
-name: Run Backend Django tests
+name: Run Django tests
 inputs:
     python-version:
         required: true
-        type: string
```

Comment:
Action inputs don't actually have a `type` parameter (they're all strings for GitHub)